ContactUs.html

```diff
@@ -10,8 +10,13 @@
 <meta name="google-signin-client_id" content="365679910149-388t7l2qc3kflnj52f01mk4bku5ffiis.apps.googleusercontent.com">
 <script src="https://apis.google.com/js/platform.js" async defer></script>
 <title>Contact Us</title>
-<script src="https://code.jquery.com/jquery-3.3.1.js"></script>
-<script src="js/NavOpacity.js"></script>
+<style type="text/css">
+.i {position: 600px;}
+.i img {
+height: 100px;
+width: 100px;
+}
+</style>
```

Comment:
Please don't change the common header. "EventBuildSystem.jar" will delete them. Rather, put these changes in ContactUs/page.css. Also, CSS in HTML is not ideal.